packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -972,6 +983,7 @@ class ImportedBucket extends BucketBase {
     this.bucketArn = parseBucketArn(this, props);
     this.bucketName = bucketName;
     this.domainName = props.bucketDomainName || this.generateDomainName();
+    this.bucketWebsiteUrl = props.bucketWebsiteUrl || this.generateBucketWebsiteUrl();
```

Comment:
I think we do. I was basing this off the current CloudFormation functionality; it will always allow you to output (and therefore read) the website URL even if web hosting is not configured.